solutions/CSES Sum of Divisors.mdx

```diff
@@ -5,23 +5,37 @@ title: Sum of Divisors
 author: Benjamin Qi
 ---
 
-Our goal is to compute the summation
-$\sum_{i=1}^n\left\lfloor \frac{n}{i}\right\rfloor\cdot i$ in faster than
-$\Theta(n)$ time.
+Try to prove that the question is equivalent to computing the summation
+$\sum_{i=1}^n\left\lfloor \frac{n}{i}\right\rfloor\cdot i$ 
 
 <Spoiler title="Hint">
 
+The first key observation is to notice that for each divisor $i$ within the 
+range $[1,N]$ will appear exactly $\lfloor\frac {n}{i}\rfloor$ times.
+This implies that each divisor will contribute $i*\lfloor\frac{n}{i}\rfloor$ 
+to the answer. Applying this to all divisors, the question is now equivalent to 
```

Comment:
This part is probably fine, but the other stuff is redundant in my opinion

Reply:
Sure should I just remove the rest and keep this ?